Sims1WidescreenPatcher.Core/ViewModels/MainWindowViewModel.cs

```diff
@@ -119,7 +119,7 @@ private bool IsBusy
     }
 
     public AvaloniaList<WrapperUtility.Wrapper> Wrappers { get; } =
-        new(WrapperUtility.Wrapper.None, WrapperUtility.Wrapper.DgVoodoo2, WrapperUtility.Wrapper.DDrawCompat);
+        new(WrapperUtility.Wrapper.DDrawCompat, WrapperUtility.Wrapper.DgVoodoo2, WrapperUtility.Wrapper.None);
```

Comment:
We'll want to add some logic that detects the operating system and changes the default wrapper based on that. I'm thinking:

* Windows: DDrawCompat,DGVoodoo2,None
* Linux: None,DgVoodoo2
* macOS: None

As it is right now, Linux and macOS default to ddrawcompat, given prompts that will not affect their setup, and will just confuse them.

Reply:
Thanks! I will look into the best way to do this in C# unless you have any ideas.

Reply:
This PR is how I'd do it https://github.com/voicemxil/TS1-Complete-Collection-Widescreen-Patcher/pull/1